modules/tinymce/src/plugins/paste/test/ts/atomic/NewlinesTest.ts

```diff
@@ -0,0 +1,60 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { assert } from 'chai';
+
+import * as Newlines from 'tinymce/plugins/paste/core/Newlines';
+
+describe('atomic.tinymce.plugins.paste.NewlinesTest', () => {
+  it('isPlainText is true for text content', () => {
+    Arr.each([
+      {
+        label: 'Basic Chrome markup (including span-wrapped tab)',
+        content: '<div><span style="white-space:pre">  </span>a</div><div><br></div><div>b</div>',
+      },
+      {
+        label: `Case shouldn't matter`,
```

Comment:
Any reason this one is in backticks?

```suggestion
        label: 'Case shouldn't matter',
```

Reply:
`couldn't` it has an apostrophe so the above would be a syntax error. This is how that was done at one point in time apparently. I'm inclined to leave it as is.

Reply:
This always gets me :P